charon/tests/generated-ml/GAstOfJson.template.ml

```diff
@@ -0,0 +1,343 @@
+(** WARNING: this file is partially auto-generated. `GAstOfJson.template.ml`
```

Comment:
This comment is slightly confusing. My understanding is that it should be read from the `GAstOfJson.ml` file, but when we read it in the `.template` file it is not clear whether it is ok to modify this file or not.